### DISCUSSION TITLE
Trades from FTX to ohlcv in timescaledb

### DISCUSSION THREAD

**Discussion by Jacquelefacque** [bmoscon/cryptofeed#763] [Q&A]
Hello! Thanks for the great project!  I'm still very green to coding. I've been trying to gather trades data FTX and write it into timescaledb and then use time_bucket to make candles, but the candles I get dont match the ones on the exchange. Tried with one or many pairs still the same. Its not the timezone problem(everything is UTC). Tried from 1 min to 1 hour and still its different. I'm 97% sure I'm doing the db part right. I'm using the default code from demo_postgres.py with this line modified: 

`f.add_feed(FTX(config="config.yaml", symbols=['SOL-USD'], channels=[TRADES], callbacks={TRADES: TradePostgres(**postgres_cfg)}))`

Could there be some kind of a problem with cryptofeed or am I doing something wrong. Can someone try to replicate this?

**Comment by bmoscon** [comment]
why not just collect the candles they provide? there are probably a number of factors here like the exchange may not report every single trade that happens, or they may not disclose how they are calculating their candles. You'd have to ask them. Try using the data from their REST api and comparing it to your data (trades from REST vs trades from websocket), is it the same? if so, take it up with FTX, if they are different you have an issue somewhere and data is being lost/missed.
